src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -227,6 +235,7 @@ public void parseOSMetrics(String rootLocation, long startTime,
 
         long mFinalT = System.currentTimeMillis();
         LOG.info("Total time taken for parsing OS Metrics: {}", mFinalT - mCurrT);
+        timingStats.put("parseOSMetrics", (double)(mFinalT - mCurrT));
```

Comment:
I didn't see that you count parseNodeMetrics.

Reply:
We can add these in the next revision. ParseNodeMetrics are per metric name, which will be little many; we can work on adding the required